$NetBSD$

According to wait(2) to use wait4() <sys/resource.h> and <sys/wait.h> should
be included.

--- src/ApvlvUtil.cpp.orig	2010-01-20 05:52:51.000000000 +0100
+++ src/ApvlvUtil.cpp
@@ -30,7 +30,8 @@
 
 #include <stdlib.h>
 #ifndef WIN32
-#include <wait.h>
+#include <sys/resource.h>
+#include <sys/wait.h>
 #endif
 #include <gtk/gtk.h>
 
